base/ext/chrono_io.h

```diff
@@ -1,19 +1,16 @@
 #pragma once
 
 #include <chrono>
-#include <ctime>
 #include <string>
-#include <iomanip>
 #include <sstream>
+#include <cctz/time_zone.h>
 
 
 namespace ext
 {
     inline std::string to_string(const std::time_t & time)
     {
-        std::stringstream ss;
-        ss << std::put_time(std::localtime(&time), "%Y-%m-%d %X");
```

Comment:
This is real issue. The `localtime` function cannot be used in multithreaded code. And also this function is extremely slow. @akuzm 

But if we have no reports from TSan then it's happened to be not used concurrently.